### PR TITLE
Call status realtime websocket service

### PR DESCRIPTION
**Pull request by mmadariaga**
A basic web socket service implementation to expose call information (https://github.com/irontec/ivozprovider/pull/1336) in realtime

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: microservices/realtime/bin/feed [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env php
+<?php
+
+set_time_limit(0);
+
+require __DIR__ . '/../src/Feeder.php';
```
Comment (mmadariaga) [inline microservices/realtime/bin/feed]:
```suggestion
require __DIR__ . '/../src/Feeder/Feeder.php';
```

File: microservices/realtime/install [hunk 2]
```diff
@@ -0,0 +1,16 @@
+$ sudo pecl install swoole-4.3.5
```
Comment (Kaian) [inline microservices/realtime/install]: This is now available as php-swoole package in extra component of our repository

File: microservices/realtime/install [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+enable http2 support? [no] : yes
+enable mysqlnd support? [no] : yes
+enable postgresql coroutine client support? [no] : no
+
+
+
+$ sudo pecl install redis
+downloading redis-5.2.0.tgz ...
+
+enable igbinary serializer support? [no] :
```
Comment (Kaian) [inline microservices/realtime/install]: This is now available as php-redis package in extra component of our repository

File: microservices/realtime/src/Services/Sentinel.php [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+                $this->master = $this->getRedisMasterOrThrowException(
+                    $config
+                );
+
+                $this->sentinel = $config;
+
+                break;
+            } catch (\Exception $e) {
+                echo "ERROR: " . $e->getMessage() . "\n";
+                continue;
```
Comment (Kaian) [inline microservices/realtime/src/Services/Sentinel.php]: continue as last for statement is redundant but you can leave it as is if you want to clarify the differences between try and catch blocks

File: microservices/realtime/src/Services/Sentinel.php [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+                $config = $this->sentinels[$i];
+                $this->master = $this->getRedisMasterOrThrowException(
+                    $config
+                );
+
+                $this->sentinel = $config;
+
+                break;
+            } catch (\Exception $e) {
+                echo "ERROR: " . $e->getMessage() . "\n";
```
Comment (Kaian) [inline microservices/realtime/src/Services/Sentinel.php]: take into account that if you use stderr for errors, journald will manage to tag them as error messages and display them in red or allow filtering with -p error.

It's not fancy but may be useful to `fwrite(STDERR, "ERROR: " . $e->getMessage() . "\n");` here.

Comment (Kaian) [reply microservices/realtime/src/Services/Sentinel.php]: Nevermind, I've just seen the logger integration in PR #1339 ;-)

**Pull request merged**